### PR TITLE
navbar shrinks when user scrolls down

### PR DESCRIPTION
**Pull request by mmoghaddam385**

### CODE REVIEW

File: website/static/css/sass/navbar.scss [hunk 1]
```diff
@@ -3,11 +3,16 @@
...
+	
+	-webkit-transition: line-height 0.5s;
+	-moz-transition: line-height 0.5s;
+	-o-transition: line-height 0.5s;
+	transition: line-height 0.5s;
 }
 
 // when the navbar collapses into a hamburger menu, move the icons to the left
-@media (max-width: 992px) {
+@media (max-width: 991px) {
```
Comment (jbonzo) [inline website/static/css/sass/navbar.scss:15]: what does this pixel change do?

File: website/static/js/scroll.js [hunk 2]
```diff
@@ -26,6 +26,46 @@ $(document).ready(function() {
...
+        $('.navbar-nav > li > a, .navbar-brand').css('height', height)
+        $('.navbar-nav > li > a, .navbar-brand').css('line-height', height)
+        $('.navbar-contact-icon').css('line-height', height)
+    }
+
+    function navbarOnScroll() {
+        var winWidth = $(window).width()
+
+        // only if the navbar isn't collapsed
+        if (winWidth >= 992) {
```
Comment (jbonzo) [inline website/static/js/scroll.js]: make a var for the 992 value?

Comment (jbonzo) [reply website/static/js/scroll.js]: ```magic number```

File: website/static/css/sass/navbar.scss (hunk 1 above)
Comment (jbonzo) [reply website/static/css/sass/navbar.scss:15]: jc

Comment (jbonzo) [inline website/static/css/sass/navbar.scss]: this is a test

**Comment by jbonzo** [comment]
I think the Icon can still be bigger. I'll play with it and push a commit to this branch

**Comment by jbonzo** [comment]
Can you let me know what you think and respond to my comments?

**Comment by mmoghaddam385** [comment]
LGTM, removed magic number but haven't tested because I don't have all the dependencies on my computer yet

**Review by jbonzo — APPROVED** [review]
Merge if you think it looks good. If you don't care too much then just merge

**Comment by mmoghaddam385** [comment]
looks good +1

**Pull request merged**